spark/src/main/kotlin/com/adevinta/spark/components/buttons/ButtonOutlined.kt

```diff
@@ -76,8 +77,12 @@ public fun ButtonOutlined(
         targetValue = intent.colors().color,
         label = "content color",
     )
+    val disabledContentColor = contentColor.copy(alpha = SparkTheme.colors.dim3)
+        .compositeOver(SparkTheme.colors.surface)
```

Comment:
We don't want the composition on the `surface` here as the button may be displayed on other surfaces than `surface`